src/components/containers/Login.js

```diff
@@ -1,26 +1,25 @@
 /*global chrome*/
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { useState } from 'react';
 
 import Navbar from '../containers/Nav';
-import Button from '../presentational/styled/Button';
+import { FullButton } from '../presentational/styled/Button';
 
 import AuthService from '../../services/AuthService';
 
-async function getUser() {
-  AuthService.signIn(function(user) {
-    console.log(user)
-    console.log("HandleLogin")
-  });
-}
+const Login = ({ handleUser }) => {
```

Comment:
What is this handleUser?

Reply:
Leftover code from working on a dynamic navbar. Has been removed in latest commit.